src/sources/commondatamodel.jl

```diff
@@ -345,7 +353,14 @@ function _cdmlookup(
         else
             boundskey = var.attrib["bounds"]
             boundsmatrix = Array(ds[boundskey])
-            Explicit(boundsmatrix), Intervals(Center())
+            locus = if all(boundsmatrix[1,:] .== index)
```

Comment:
```suggestion
            locus = if mapredreduce(==, &, view(boundsmatrix, 1, :), index)
```

Something like this is less allocs and compilation than the getindex and broadcast. Similar below.

Reply:
Yes thanks. I changed both lines.